packages/bits/src/lib/wizard-v2/wizard.component.less

```diff
@@ -34,31 +34,42 @@
     }
 }
 
+.nui-step-header-wrapper{
+    position: relative;
+}
+
 .nui-wizard-horizontal-line {
-    border-top-width: @wizard-line-width;
+    border-top-width: @nui-line-medium;
     border-top-style: solid;
     .setCssVariable(border-color, nui-color-line-default);
 
-    flex: auto;
-    height: 0;
-    margin: 0 (@wizard-line-gap - @wizard-side-gap);
-    min-width: (@wizard-line-gap + @wizard-side-gap);
+    &.nui-wizard-horizontal-line-completed{
```

Comment:
This class and associated styling should live in the less file for wizard-horizontal